apps/dashboard/src/@/api/support.ts

```diff
@@ -0,0 +1,354 @@
+"use server";
+import "server-only";
+import { NEXT_PUBLIC_THIRDWEB_API_HOST } from "@/constants/public-envs";
+import { getAuthToken, getAuthTokenWalletAddress } from "./auth-token";
+
+export interface SupportTicket {
+  id: string;
+  status: "needs_response" | "in_progress" | "on_hold" | "closed" | "resolved";
+  createdAt: string;
+  updatedAt: string;
+  messages?: SupportMessage[];
+}
+
+interface SupportMessage {
+  id: string;
+  content: string;
+  createdAt: string;
+  timestamp: string;
+  author?: {
+    name: string;
+    email: string;
+    type: "user" | "customer";
+  };
+}
+
+interface CreateSupportTicketRequest {
+  message: string;
+  teamSlug: string;
+  title: string;
+  conversationId?: string;
+}
+
+interface SendMessageRequest {
+  ticketId: string;
+  teamSlug: string;
+  message: string;
+}
+
+export async function getSupportTicketsByTeam(
```

Comment:
This should not be a server action at all - This is called in the backend. Move it outside of this file ( This file has `"use server" directive added on top )